front/components/assistant/AssistantDetails.tsx

```diff
@@ -125,16 +123,22 @@ export function AssistantDetails({
             disabled={isUpdatingScope}
             setNewScope={(scope) => updateScope(scope)}
           />
+          <AssistantListActions
+            agentConfiguration={agentConfiguration}
+            owner={owner}
+            isParentHovered={true}
+            onAssistantListUpdate={() => void mutateAgentConfigurations?.()}
+          />
         </div>
         <div>
           <AssistantEditionMenu
             agentConfigurationId={agentConfiguration.sId}
             owner={owner}
             variant="button"
-            tryButton={showTryButtonInMenu}
+            tryButton
```

Comment:
Shouldn't we gate this behind having access to large models like we do in 👇 ?
https://github.com/dust-tt/dust/blob/23ada908c01c66d03a9a15c194c8611916966b6a/front/components/assistant/GalleryAssistantPreviewContainer.tsx#L46-L49

Reply:
it's not possible at this point to have access to large models via the try

Reply:
Seen IRL, this menu **is not** displayed for global agents.